RyuSocks/Packets/Auth/UsernameAndPassword/UsernameAndPasswordResponse.cs

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) RyuSOCKS
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+using System.Security.Authentication;
+
+namespace RyuSocks.Packets.Auth.UsernameAndPassword
+{
+    public class UsernameAndPasswordResponse : Packet
+    {
+
+        public UsernameAndPasswordResponse()
+        {
+            Bytes = new byte[2];
+        }
+
+        public UsernameAndPasswordResponse(byte[] packetBytes, byte version, byte status)
+        {
+            Bytes = packetBytes;
+
+            Version = version;
+            Status = status;
+        }
```

Comment:
There should be one constructor that just takes `packetBytes`, so it can be used to easily parse incoming packets.

Could you either remove `version` and `status` from this constructor or add another one that has only one parameter for `packetBytes`?